docs/_rpc/ns-admin.md

```diff
@@ -0,0 +1,249 @@
+---
+title: admin Namespace
+sort_key: C
+---
+
+The `admin` API gives you access to several non-standard RPC methods, which will allow you to have
+a fine grained control over your Geth instance, including but not limited to network peer and RPC
+endpoint management.
+
+* TOC
+{:toc}
+
+### admin_addPeer
+
+The `addPeer` administrative method requests adding a new remote node to the list of tracked static
+nodes. The node will try to maintain connectivity to these nodes at all times, reconnecting every
+once in a while if the remote connection goes down.
+
+The method accepts a single argument, the [`enode`](https://github.com/ethereum/wiki/wiki/enode-url-format)
+URL of the remote peer to start tracking and returns a `BOOL` indicating whether the peer was accepted
+for tracking or some error occurred.
+
+| Client  | Method invocation                              |
+|:--------|------------------------------------------------|
+| Go      | `admin.AddPeer(url string) (bool, error)`      |
+| Console | `admin.addPeer(url)`                           |
+| RPC     | `{"method": "admin_addPeer", "params": [url]}` |
+
+#### Example
+
+```javascript
+> admin.addPeer("enode://a979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c@52.16.188.185:30303")
+true
+```
+
+### admin_datadir
+
+The `datadir` administrative property can be queried for the absolute path the running Geth node
+currently uses to store all its databases.
+
+| Client  | Method invocation                 |
+|:--------|-----------------------------------|
+| Go      | `admin.Datadir() (string, error`) |
+| Console | `admin.datadir`                   |
+| RPC     | `{"method": "admin_datadir"}`     |
+
+#### Example
+
+```javascript
+> admin.datadir
+"/home/karalabe/.ethereum"
```

Comment:
```suggestion
"/home/johndoe/.ethereum"
```

Reply:
A more generic option is also

```suggestion
"~/.ethereum"
```

Reply:
I'm afraid javascript won't figure out your home dir and replace it with `~` in the path.